src/Log/Minecraft/Vanilla/NeoForge/NeoForgeCrashReportLog.php

```diff
@@ -22,6 +22,9 @@ public static function getDetectors(): array
             (new MultiPatternDetector())
                 ->addPattern("/^---- Minecraft Crash Report ----$/m")
                 ->addPattern("/^\tNeoForge: net\.neoforged:/m"),
+            (new MultiPatternDetector())
+                ->addPattern("/^---- Minecraft Crash Report ----$/m")
+                ->addPattern("/^\t\t\S*\s*\|\S*\s*\|neoforge\s*\|[0-9\.]+(?:-beta)?\s*\|/m"),
```

Comment:
I think this could be a little less specific, how about `/^\t\t[^|]*\|[^|]*\|\s*neoforge\s*\|/m`?
This also doesn't break when NeoForge eventually adds a space after the | because they notice that it looks weird.

Reply:
Okay, but I'd still expect two more columns for the version information and the `Manifest: NOSIGNATURE`.
What do you think about `/^\t\t[^|]*\|[^|]*\|\s*neoforge\s*\|[^|]*\|[^|]*/m`?

Reply:
I see no reason to match for these columns. That just means this is more likely to break when neoforge eventually changes this table

Reply:
Done